### PR TITLE
Responsive sidebar

### PR DESCRIPTION
**Pull request by msukkar**
@aw3as New consolidated PR

### CODE REVIEW

File: app/assets/javascripts/scheduler.js [hunk 1]
```diff
@@ -3,8 +3,13 @@
...
 // $ denotes jQuery - $(document) means we select the "document" or HTML page object
 // We attach an anonymous function to be executed when the page is "ready" - all DOM elements are loaded
+
+$(window).resize(function(){
+	$('#calendar').css('width', $('#search-bar').parent().css('width').split('p')[0] - $('#search-bar').css('width').split('p')[0] - 20);
+});
+
 $(document).ready(function() {
-	$('#calendar').css('width', $('#search-bar').parent().css('width').split('p')[0] - $('#search-bar').css('width').split('p')[0] - 40);
+	$('#calendar').css('width', $('#search-bar').parent().css('width').split('p')[0] - $('#search-bar').css('width').split('p')[0] - 20);
```
Comment (aw3as) [inline app/assets/javascripts/scheduler.js]: wot dis

Comment (msukkar) [inline app/assets/javascripts/scheduler.js]: changed width between calendar and search

**Comment by aw3as** [comment]
rewrite css media queries using new bootstrap screen size delimiters now that you're using their variables

File: app/assets/stylesheets/welcome.css.scss [hunk 2]
```diff
@@ -31,4 +32,10 @@
 
 .user-sign-up {
   color: white;
+}
+
+@media (max-width: $screen-md){
```
Comment (aw3as) [inline app/assets/stylesheets/welcome.css.scss]: bootstrap class this inside of css

File: app/assets/stylesheets/welcome.css.scss [hunk 3]
```diff
@@ -1,4 +1,5 @@
 @import "variables";
+@import "bootstrap/variables";
```
Comment (aw3as) [inline app/assets/stylesheets/welcome.css.scss]: is this line needed?

**Pull request merged**